linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/oval/shared.xml

```diff
@@ -17,9 +17,13 @@
   </ind:textfilecontent54_test>
 
   <ind:textfilecontent54_object id="object_accounts_passwords_pam_tally2_even_deny_root" comment="Check even deny root configuration of pam_tally2" version="1">
+    {{% if product in ["sle12","sle15"] %}}
```

Comment:
The rule is restricted to sle products. So, why the conditionals here? Would be simpler to only update the file.